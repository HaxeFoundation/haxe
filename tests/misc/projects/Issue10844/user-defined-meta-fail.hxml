--macro haxe.macro.Compiler.registerCustomMetadata({metdata: ':zzz', doc: 'something'}, 'myapp')
--help-user-metas
