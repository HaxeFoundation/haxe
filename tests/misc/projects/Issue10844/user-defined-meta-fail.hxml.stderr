(unknown) : Object requires field metadata
(unknown) : ... For function argument 'meta'
